OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -3206,4 +3212,7 @@ private static boolean isValidOutcomeEntry(String name) {
     * End OneSignalOutcome module
     */
 
+   interface OperationCompletedCallback {
+      void completed(boolean result);
```

Comment:
Is the `boolean result` necessary?

Reply:
If so change name to `accepted`

Reply:
no rly, just wanted to be like on iOS, what do you think @mikechoch ?

Reply:
its fine, just rename it so its like iOS